common-queries/src/main/scala/queries/schemas/odb/implicits.scala

```diff
@@ -4,97 +4,91 @@
 package queries.schemas
 
 import clue.data.Input
-import clue.data.syntax._
+import clue.data.syntax.*
 import eu.timepit.refined.types.numeric.PosBigDecimal
 import eu.timepit.refined.types.numeric.PosLong
 import eu.timepit.refined.types.string.NonEmptyString
 import explore.model.ResizableSection
-import io.circe.syntax._
+import io.circe.syntax.*
 import lucuma.core.enums.Band
-import lucuma.core.math.BrightnessUnits._
-import lucuma.core.math._
-import lucuma.core.math.dimensional._
-import lucuma.core.model.ExposureTimeMode._
-import lucuma.core.model.ProposalClass._
-import lucuma.core.model._
-import lucuma.core.syntax.time._
+import lucuma.core.math.BrightnessUnits.*
+import lucuma.core.math.*
+import lucuma.core.math.dimensional.*
+import lucuma.core.model.ExposureTimeMode.*
+import lucuma.core.model.ProposalClass.*
+import lucuma.core.model.*
+import lucuma.core.syntax.time.*
 import lucuma.schemas.ObservationDB.Enums.PosAngleConstraintType
-import lucuma.schemas.ObservationDB.Types._
+import lucuma.schemas.ObservationDB.Types.*
 import queries.schemas.UserPreferencesDB.Types.ExploreResizableWidthInsertInput
 
 import scala.collection.immutable.SortedMap
+import scala.annotation.targetName
+import explore.model.ScienceModeBasic
+import explore.model.ScienceModeAdvanced
+import explore.model.ScienceMode
 
-final case class WidthUpsertInput(user: User.Id, section: ResizableSection, width: Int)
+case class WidthUpsertInput(user: User.Id, section: ResizableSection, width: Int)
 
 // TODO Move to lucuma-schemas
 object implicits {
-  implicit class ObsIdOps(val id: Observation.Id) extends AnyVal {
+  extension (id: Observation.Id)
     def toWhereObservation: WhereObservation =
       WhereObservation(id = WhereOrderObservationId(EQ = id.assign).assign)
-  }
 
-  implicit class ObsIdListOps(val ids: List[Observation.Id]) extends AnyVal {
+  extension (ids: List[Observation.Id])
+    @targetName("ObservationId_toWhereObservation")
```

Comment:
Isn't this only relevant to java users?

Reply:
Well, yes. But, it also seems to be a workaround for having extension methods with the same name. `toWhereObservation` in this case. It doesn't seem like it should be necessary since the actual generated methods have the type which is being extended as a parameter. But, maybe that overload is not sufficient for java-land.

Reply:
It doesn't compile otherwise. I think it's because the name and type are the same after erasure.